### PR TITLE
Add gemfury instructions

### PR DESCRIPTION
**Pull request by vitaliiprodan**
Why:

* Provide all required information about gemfury configuration

How:

* Created new md file

### CODE REVIEW

File: guides/ruby-gems/gemfury.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Gemfury usage
```
Comment (tom-mullen) [inline guides/ruby-gems/gemfury.md]: # Gemfury Usage

File: guides/ruby-gems/gemfury.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Alternative way to use private gems on multiple projects and add them to dependencies is publishing on hosted repository [gemfury](https://manage.fury.io/dashboard/dvelp).
```
Comment (tom-mullen) [inline guides/ruby-gems/gemfury.md]: [Gemfury](https://manage.fury.io/dashboard/dvelp) is a private gem server that we use to easily manage access to private gems.

File: guides/ruby-gems/gemfury.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Gem publishing
```
Comment (tom-mullen) [inline guides/ruby-gems/gemfury.md]: ## Publishing a Gem

File: guides/ruby-gems/gemfury.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+To publish new gem or new version of the gem you need to do following steps:
```
Comment (tom-mullen) [inline guides/ruby-gems/gemfury.md]: To publish new gem or new version of the gem you need to follow the steps below:

- Install gemfury gem: `gem install gemfury`
- Authenticate with your Gemfury login credentials:  `fury login`
- Build the gem: `gem build <your_gem_name>.gemspec`
- Upload your package: `fury push <your_gem_name>-<your_gem_version>.gem --as=dvelp`

That’s it!

File: guides/ruby-gems/gemfury.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+- Install gemfury gem: `gem install gemfury`
+- Authenticate with your Gemfury login credentials:  `fury login`
+- Build the gem: `gem build <your_gem_name>.gemspec`
+- Upload your package: `fury push <your_gem_name>-<your_gem_version>.gem --as=dvelp`
+
+That’s it!
+
+## Gem installing
```
Comment (tom-mullen) [inline guides/ruby-gems/gemfury.md]: ## Installing a Gem

File: guides/ruby-gems/gemfury.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+To install gem from gemfury repository you need to follow next steps
```
Comment (tom-mullen) [inline guides/ruby-gems/gemfury.md]: To install a gem from gemfury you need to follow the steps below:

- Copy your personal access token from [here](https://manage.fury.io/manage/dvelp/tokens).
- Configure your bundler: `bundle config gem.fury.io <your_token>`
- For builds and deployment, you may configure authentication through the environment: `export BUNDLE_GEM__FURY__IO=<your_token>`
- Add new source to your Gemfile: `source 'https://gem.fury.io/dvelp'`
- Bundle!

File: guides/ruby-gems/gemfury.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+- Go to https://manage.fury.io/manage/dvelp/tokens and copy your personal access token
+- Configure your bundler with `bundle config gem.fury.io <your_token>`
+- For builds and deployment, you may configure authentication through the environment: `export BUNDLE_GEM__FURY__IO=<your_token>`
+- Add new source to your Gemfile: `source 'https://gem.fury.io/dvelp'`
+- Install your gem
+
+## Configure authentication on CircleCI and heroku
```
Comment (tom-mullen) [inline guides/ruby-gems/gemfury.md]: ### Configure Authentication on CircleCI and Heroku

File: guides/ruby-gems/gemfury.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+To allow Circle CI and Heroku to install gems from gemfury you simple need to set your auth token to new environment variable: `BUNDLE_GEM__FURY__IO: <your_token>`
+
+## Gem publishing via Cicrle CI
```
Comment (tom-mullen) [inline guides/ruby-gems/gemfury.md]: ### Publishing via gem via Cicrle CI
I would also move this under the 'Publishing a Gem section'

File: guides/ruby-gems/gemfury.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+To automate gem publishing process you can add special scenario on Cicrle CI. For this add following to circle.yml file:
```
Comment (tom-mullen) [inline guides/ruby-gems/gemfury.md]: To have the gem automatically published when a new version is created, add the following to your `circle.yml` file:

**Review by tom-mullen — CHANGES_REQUESTED** [review]

**Comment by vitaliiprodan** [comment]
@tom-mullen fixed

File: guides/ruby-gems/gemfury.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+- Configure your bundler: `bundle config gem.fury.io <your_token>`
+- For builds and deployment, you may configure authentication through the environment: `export BUNDLE_GEM__FURY__IO=<your_token>`
+- Add new source to your Gemfile: `source 'https://gem.fury.io/dvelp'`
+- Bundle!
+
+### Configure Authentication on CircleCI and Heroku
+
+To allow Circle CI and Heroku to install gems from gemfury you simple need to set your auth token to new environment variable: `BUNDLE_GEM__FURY__IO: <your_token>`
+
+## Other links
```
Comment (tom-mullen) [inline guides/ruby-gems/gemfury.md]: Other Links

File: guides/ruby-gems/gemfury.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+To publish new gem or new version of the gem you need to follow the steps below:
+
+- Install gemfury gem: `gem install gemfury`
+- Authenticate with your Gemfury login credentials:  `fury login`
+- Build the gem: `gem build <your_gem_name>.gemspec`
+- Upload your package: `fury push <your_gem_name>-<your_gem_version>.gem --as=dvelp`
+
+That’s it!
+
+## Publishing via gem via Cicrle CI
```
Comment (tom-mullen) [inline guides/ruby-gems/gemfury.md]: ### Publishing via Cicrle CI

**Review by tom-mullen — CHANGES_REQUESTED** [review]

**Review by tom-mullen — APPROVED** [review]

**Pull request merged**